.github/workflows/check-tests.yml

```diff
@@ -0,0 +1,14 @@
+name: Pre-Commit Hook
+
+on:
+  pull_request:
+    paths:
+      - 'apps/**'
+      - 'libs/**'
```

Comment:
Can we make it run only on .e2e, .test, .spec files changes and not on each request?